setup.cfg

```diff
@@ -22,4 +22,9 @@ addopts = -rsxX --tb=native -p no:django
 norecursedirs = .git docs config docker __pycache__
 testpaths = socorro/
 # Transform all warnings into errors
-filterwarnings = error
+filterwarnings =
+    error
+    # grpcio kicks this up for some reason in pytest
+    ignore::ImportWarning
+    ignore:::grpcio[.*]
+    ignore:::google[.*]
```

Comment:
These packages kick up a bunch of warnings. We have pytest set so that warnings -> errors. In this case, there's nothing we can do here, so we should just silence them for now.